package.json

```diff
@@ -14,7 +14,7 @@
     "lint:css": "stylelint \"**/*.scss\"",
     "lint": "yarn lint:js && yarn lint:css",
     "lint:fix": "yarn prettier && eslint --fix \"src/**/*.+(js|jsx|ts|tsx)\" && stylelint --fix \"**/*.scss\"",
-    "type-css": "yarn typed-scss-modules src/ --exportType default --watch --listDifferent",
+    "type-css": "yarn typed-scss-modules src/ --exportType default",
```

Comment:
Removed since it didn't seem to be working, we just rebuild our SCSS typings by running dev again instead